app/models/spree/order.rb

```diff
@@ -141,6 +141,11 @@ def states
         joins('LEFT OUTER JOIN spree_products ON (spree_products.id = spree_variants.product_id)')
     }
 
+    scope :placed_by_customer_email, lambda { |email|
+      joins('JOIN customers ON (customers.id = spree_orders.customer_id)').
```

Comment:
Even easier:
```suggestion
      joins(:customer).
```

I compared the two:

```rb
Spree::Order.placed_by_customer_email(nil).to_sql
=> "SELECT \"spree_orders\".* FROM \"spree_orders\" JOIN customers ON (customers.id = spree_orders.customer_id) WHERE (customers.email = NULL)"

Spree::Order.joins(:customer).where(customer: {email: nil}).to_sql
=> "SELECT \"spree_orders\".* FROM \"spree_orders\" INNER JOIN \"customers\" \"customer\" ON \"customer\".\"id\" = \"spree_orders\".\"customer_id\" WHERE \"customer\".\"email\" IS NULL"
```

But I have another idea I'll lay out further down.